lib/broccoli_sass_compiler.js

```diff
@@ -485,7 +485,7 @@ BroccoliSassCompiler.prototype.handleCacheHit = function(details, inputAndOutput
 BroccoliSassCompiler.prototype.scopedFileName = function(file) {
   file = this.relativize(file);
   if (this.treeName) {
-    return this.treeName + "/" + file;
+    return this.treeName + path.sep + file;
```

Comment:
Is there anything we can do to make this happen?  This is a bit of a show stopper on Windows!